content/en/profiler/guide/isolate-outliers-in-monolithic-services.md

```diff
@@ -0,0 +1,94 @@
+
+## Comparing flame graphs
+
+Now that the graph shows data for the right time and endpoint, you probably have enough data to understand what is causing the spike in CPU utilization. But if not, you can compare the flame graph for the spike with a time when utilization was more acceptable.
+
+To see if there are differences in which methods are using a lot of CPU time between a spike and normal usage, click **Compare** (on the far right side of the search field) and select `Previous 15 minutes`. This opens the Comparison view. 
+
+The view shows two graphs, labeled `A` and `B`, each representing a time range for CPU utilization per call for `GET /store_history`. Adjust the time selector for `A` so that it is scoped to a period with low CPU utilization per call:
+
+{{< img src="profiler/guide-monolithic-outliers/5-outliers-monolith-compare-flame-graphs.png" alt="Your image description" style="width:100%;" >}}
+
+The comparison shows the different methods causing CPU utilization on the right side (timeframe `B` during the spike) that are not used at all in timeframe `A` (normal CPU). The culprit, `Product.loadAssets(int)`, is causing the spikes. 
+
+To fix the problem, optimize the method. Looking at the method code, the signature is `Product(int id, String name, boolean shouldLoadAssets)` and we do not need to load assets for the response to the `GET /store_history` endpoint. This implies that there is a bug further up the call stack that improperly instructs the `Product` constructor to load assets.
+
+Fix that bug and verify that the spikes go away, using the timeseries graphs discussed earlier.
+
+## Isolate the impact of operations (Java)
+
+There are other attributes available in the profiler. For example, you can filter and group a flame graph by operation names, rather than by functions or threads. For a monolithic application this can be more helpful, as it can more directly state which resources are utilizing more CPU, even if they are shared between endpoints.
+
+The APM `Trace operation` attribute lets you filter and group a flame graph with the same granularity as what's in the traces for the selected endpoints. This is a good middle between the high granularity of threads or methods, and the low granularity of entire endpoints. To isolate operations, select `Trace Operation` from the **CPU time by** drop-down: 
+
+{{< img src="profiler/guide-monolithic-outliers/7-outliers-monolith-trace-operation.png" alt="Your image description" style="width:100%;" >}}
+
+Now you can see that the `ModelTraining` operation is taking more CPU time than its primary use in the `GET /train` endpoint, so it must be used in other places. Select it to find out where else it is used. In this case, `ModelTraining` is also use by `POST /update_model`.
```

Comment:
```suggestion
Now you can see that the `ModelTraining` operation is taking more CPU time than its primary use in the `GET /train` endpoint, so it must be used in other places. Click on it to find out where else it is used. In this case, `ModelTraining` is also use by `POST /update_model`.
```

We had the feedback that the summary table is not fully intuitive, let's clarify what the exact action is